tests/test_wordfreq_build.py

```diff
@@ -0,0 +1,47 @@
+import pytest
+from shutil import rmtree
+
+from tests.testing_utils import (
+    gzipped_result_dir_same_as_reference,
+    result_dir_same_as_reference,
+    run_snakemake,
+)
+
+
+@pytest.fixture(scope='session')
+def run_build(test_env_variables, setup_input_files):
+    run_snakemake(test_env_variables)
+    yield
+    rmtree('tests/data/extracted')
```

Comment:
Could this list of directory names be abstracted away somewhere? It seems to be repeated a lot.

Reply:
I can do that.